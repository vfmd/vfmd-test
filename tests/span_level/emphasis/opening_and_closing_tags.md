No emphasis in the following:

one *two three *four

one* two three* four

one* two three *four

one **two three **four

one** two three** four

one** two three **four

one _two three _four

one_ two three_ four

one_ two three _four

one __two three __four

one__ two three__ four

one__ two three __four
