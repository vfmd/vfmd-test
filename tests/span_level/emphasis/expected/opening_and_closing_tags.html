<p>No emphasis in the following:</p>

<p>one *two three *four</p>

<p>one* two three* four</p>

<p>one* two three *four</p>

<p>one **two three **four</p>

<p>one** two three** four</p>

<p>one** two three **four</p>

<p>one _two three _four</p>

<p>one_ two three_ four</p>

<p>one_ two three _four</p>

<p>one __two three __four</p>

<p>one__ two three__ four</p>

<p>one__ two three __four</p>
